luigi/configuration.py

```diff
@@ -33,11 +33,11 @@
 import os
 import warnings
 
-from configparser import Interpolation
 try:
     from ConfigParser import ConfigParser, NoOptionError, NoSectionError
+    Interpolation = object
```

Comment:
What's going on here? Was this copied from Luigi upstream?

Reply:
yeah must be but looks like its not needed so going to remove it